configs/config.yml

```diff
@@ -13,3 +13,6 @@ preprocessing:
       img_size:
         - 48
         - 48
+logging:
+  logging_dir: /Users/dvand/garbage/logs
+  logging_config_file: /Users/dvand/git/wai_data_tools/configs/default_logging_config.conf
```

Comment:
maybe commit non user specific paths?

Reply:
woops good find, this is a mistake, it should be the default values.

Reply:
fixed